pkg/gofr/crud_handlers_test.go

```diff
@@ -144,13 +147,8 @@ func Test_CreateHandler(t *testing.T) {
 	}
 }
 
+//nolint:funlen // test fails when used 2 loops in test cases approach
```

Comment:
we shouldn't use `nolint:funlen`. As discussed you can break the test into `Test_GetHandler_MySQL` and `Test_GetHandler_Postgres`, i know there will be some code repetition (try extracting out the common code like making of context, db , metrics etc. if required ). we can prefer that over using nolints.

Reply:
I'm not sure why it was failing for me for some strange reason - that's why I avoided 2 loops approach (seems to be kind of local issue), but now I tried once again and it works.
So I propose to stay in this format:

```
for dialect : dialects {
     for testCase : testCases {
         // test body
     } 
}
```

It will help us avoid code duplication and reduce costs when we decide to add a new dialect.